### PR TITLE
Add debug for cake app authorisation issue

### PR DESCRIPTION
**Pull request by stearagon**
tin

### CODE REVIEW

**Comment by stearagon** [comment]
Hey @davegriffiths. I just added a common issue to the readme that we have been running into. Review?

**Comment by davegriffiths** [comment]
Nice. 

Even better would be to fix cake so it spits out an error message out when you're not authorised.

**Comment by davegriffiths** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -32,3 +32,8 @@ The name of the local database you want to restore to
 
 The name of the app where the database is located. For Example, if the app URL was cake.herokuapp.com you would set the following.
 - ``` app: cake ```
+
+### Debugging
```
Comment (davegriffiths) [inline README.md]: Maybe change this to common issues?

**Comment by stearagon** [comment]
@davegriffiths changed the heading to common issues. I think making it spit out an error message will have to wait another day. Feel free to merge.

**Comment by toreriklinnerud** [comment]
Hah, for a second I thought you had added code to recognise the issue and warn about it. But this is a start :+1:

**Pull request merged**